SpatialGDK/Source/SpatialGDKEditorToolbar/Public/SpatialGDKSimulatedPlayerDeployment.h

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Input/Reply.h"
+#include "Layout/Visibility.h"
+#include "SpatialGDKEditor.h"
+#include "SpatialGDKEditorCloudLauncherSettings.h"
+#include "Templates/SharedPointer.h"
+#include "Widgets/DeclarativeSyntaxSupport.h"
+#include "Widgets/Input/SEditableTextBox.h"
+#include "Widgets/Layout/SBorder.h"
+#include "Widgets/SCompoundWidget.h"
+
+class SWindow;
+
+enum class ECheckBoxState : uint8;
+
+class SSpatialGDKSimulatedPlayerDeployment : public SCompoundWidget
+{
+public:
+
+	SLATE_BEGIN_ARGS(SSpatialGDKSimulatedPlayerDeployment) {}
+
+	/** A reference to the parent window */
+	SLATE_ARGUMENT(TSharedPtr<SWindow>, ParentWindow)
+	SLATE_ARGUMENT(TSharedPtr<FSpatialGDKEditor>, SpatialGDKEditor)
+
+	SLATE_END_ARGS()
+
+public:
+
+	void Construct(const FArguments& InArgs);
+
+private:
+	/** The parent window of this widget */
+	TWeakPtr<SWindow> ParentWindowPtr;
+
+	/** Pointer to the SpatialGDK editor */
+	TWeakPtr<FSpatialGDKEditor> SpatialGDKEditorPtr;
+
+	/** Pointer to the Spatial GDK Cloud Settings */
+	TSharedPtr<USpatialGDKEditorCloudLauncherSettings> SpatialGDKCloudSettings;
```

Comment:
Why do we store a reference to the settings when we tend to get it using MutableDefault<> in the cpp file?

Reply:
Good catch - removed the reference from the file 👍